src/main/fc/rc.c

```diff
@@ -572,55 +567,66 @@ FAST_CODE_NOINLINE void calculateFeedforward(const pidRuntime_t *pid, int axis)
     // smooth the setpointSpeed value
     setpointSpeed = prevSetpointSpeed[axis] + pid->feedforwardSmoothFactor * (setpointSpeed - prevSetpointSpeed[axis]);
 
-    // calculate acceleration and attenuate
+    // calculate acceleration from smoothed setpointSpeed and attenuate
     setpointAcceleration = (setpointSpeed - prevSetpointSpeed[axis]) * rxRate * 0.01f;
     prevSetpointSpeed[axis] = setpointSpeed;
 
-    // smooth the acceleration element (effectively a second order filter) and apply jitter reduction
+    // smooth the acceleration element (effectively a second order filter since incoming setpoint was smoothed)
+    // apply jitter reduction to acceleration here, effectively twice on acceleration
     setpointAcceleration = prevAcceleration[axis] + pid->feedforwardSmoothFactor * (setpointAcceleration - prevAcceleration[axis]);
     prevAcceleration[axis] = setpointAcceleration * zeroTheAcceleration;
     setpointAcceleration = setpointAcceleration * pid->feedforwardBoostFactor * jitterAttenuator * zeroTheAcceleration;
 
-    if (axis == FD_ROLL) {
-        DEBUG_SET(DEBUG_FEEDFORWARD, 1, lrintf(setpointSpeed * 0.1f)); // base feedforward without acceleration
-    }
+    feedforward = setpointSpeed + setpointAcceleration;
 
-    float feedforward = setpointSpeed + setpointAcceleration;
+    // apply jitter attenuation to classic feedforward elements only (twice on acceleaertion)
+    feedforward *= jitterAttenuator;
```

Comment:
(maybe apply jitterAttenuator to setpointSpeed  and SQ(jitterAttenuator) to setpointAcceleration explicitly (before sum)

Reply:
Thanks.
Yes that's perhaps a clearer code solution.
But isn't this less maths, since we only do two multiplications by jitterAttenuator? Whereas with your proposal we do a square and a multiplication?